worlds/AutoWorld.py

```diff
@@ -498,12 +498,24 @@ def create_filler(self) -> "Item":
     def get_location(self, location_name: str) -> "Location":
         return self.multiworld.get_location(location_name, self.player)
 
+    def get_locations(self) -> "Iterable[Location]":
+        return self.multiworld.get_locations(self.player)
+
     def get_entrance(self, entrance_name: str) -> "Entrance":
         return self.multiworld.get_entrance(entrance_name, self.player)
 
+    def get_entrances(self) -> "Iterable[Entrance]":
+        return self.multiworld.get_entrances(self.player)
+
     def get_region(self, region_name: str) -> "Region":
         return self.multiworld.get_region(region_name, self.player)
 
+    def get_regions(self) -> "Iterable[Region]":
+        return self.multiworld.get_regions(self.player)
+
+    def add_to_starting_inventory(self, item: Item) -> None:
```

Comment:
Given that this doesn't _actually_ add to `start_inventory`, I think using `push_precollected` for the name made more sense.

Reply:
Well, I have heard an opinion both ways now so not sure what to do :D